SalesforceCommerceCloud/pwa-kit
# A11y: Fix breadcrumbs and product edit modal (@W-12627097@)

# Description

This PR fixes some accessibility issues:

### 1. [Accessibility][Breadcrumb] Hide icons from screenreader as they are decorative
W-12627097 & W-12627169

Problem Description: Bread crumb chevrons should not be read out using a screen reader

Files edited:
- `packages/template-retail-react-app/app/components/breadcrumb/index.jsx`

### 2. [Accessibility][Product View Modal] Provide a descriptive dialog title for ProductView Modal
W-12627130

Problem description: There is not any good descriptive text describing the edit modal, making the purpose of the modal unclear for users with screen readers

Files edited:
- `packages/template-retail-react-app/app/components/product-view-modal/bundle.jsx`
- `packages/template-retail-react-app/app/components/product-view-modal/index.jsx`



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

`git checkout ju/a11y-breadcrumbs`

To test breadcrumbs:

1. Navigate to a product's PDP page
2. Enable VoiceOver for MacOS
3. Focus on the first link of the breadcrumb and press `Control + Option + A`
4. Observe that the chevron is not called out

To test edit modal:

1. Add a product to cart
2. Add a product bundle to cart
3. Navigate to cart page
4. Using a screen reader, click edit on either product line item
5. Observe read out `Edit Modal for <product-name>`

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
